tailwind.config.js

```diff
@@ -1,34 +1,47 @@
 /** @type {import('tailwindcss').Config} */
-import { nextui } from '@nextui-org/react';
+import { nextui } from "@nextui-org/react";
 
 module.exports = {
   content: [
     "./app/**/*.{js,ts,jsx,tsx,mdx}",
     "./pages/**/*.{js,ts,jsx,tsx,mdx}",
     "./components/**/*.{js,ts,jsx,tsx,mdx}",
- 
+
     // Or if using `src` directory:
     "./src/**/*.{js,ts,jsx,tsx,mdx}",
     "./node_modules/@nextui-org/theme/dist/**/*.{js,ts,jsx,tsx}",
   ],
   theme: {
     extend: {
       fontFamily: {
-        poppins: ['var(--font-poppins)'],
-        display: ['var(--font-noto-serif-display)'],
+        poppins: ["var(--font-poppins)"],
+        display: ["var(--font-noto-serif-display)"],
       },
       colors: {
-        "primary": "#1F4776",
+        primary: "#1F4776",
         "a-navy": "#1F4776",
         "a-black": "#393E46",
         "a-pink": "#FCF0F2",
         "a-green": "#2A9E2F",
         "a-red": "#9E2A2A",
       },
+      screens: {
+        sm: "325px",
+        md: "525px",
+        lg: "812px",
+      },
+      fontSize: {
```

Comment:
nothing wrong but i'm just wondering why fontSize needs to be added because i think the sizing you put is the same as the tailwind default ones?